spec.bs

```diff
@@ -1172,6 +1172,10 @@ To <dfn>validate and convert auction ad config</dfn> given an {{AuctionAdConfig}
 1. If |config|["{{AuctionAdConfig/sellerTimeout}}"] [=map/exists=], set |auctionConfig|'s
   [=auction config/seller timeout=] to |config|["{{AuctionAdConfig/sellerTimeout}}"] in milliseconds
   or 500 milliseconds, whichever is smaller.
+1. Let |reportingTimeout| be |config|["{{AuctionAdConfig/reportingTimeout}}"] if it [=map/exists=],
+  50 milliseconds otherwise.

```

Comment:
```suggestion
  50 otherwise.

```

We don't need a unit. The type is `unsigned long long` which doesn't know anything about milliseconds.